code/game/gamemodes/miniantags/abduction/abduction_surgery.dm

```diff
@@ -14,12 +14,41 @@
 	requires_organic_bodypart = TRUE
 	requires_bodypart = TRUE
 
-/datum/surgery/organ_extraction/can_start(mob/user, mob/living/carbon/target, target_zone, obj/item/tool,datum/surgery/surgery)
+/datum/surgery/organ_extraction/can_start(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
 	var/mob/living/carbon/human/H = user
 	// You must either: Be of the abductor species, or contain an abductor implant
-	if((isabductor(H) || (locate(/obj/item/bio_chip/abductor) in H)))
-		return TRUE
-	return FALSE
+	if(!isabductor(H) && !(locate(/obj/item/bio_chip/abductor) in H))
+		return FALSE
+
+	if(HAS_TRAIT(target, TRAIT_NO_BONES))
+		return FALSE
+
+	return TRUE
+
+/datum/surgery/organ_extraction_boneless
```

Comment:
Why not make this a subtype of `/datum/surgery/organ_extraction`?

Reply:
Everything would have to be overridden anyways, and this keeps consistency with the existence of organ_manipulation_boneless.